atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -600,24 +601,23 @@ private void sanityCheckRingConsistency() {
 
             RuntimeException ex = new IllegalStateException("Hosts have differing ring descriptions."
                     + " This can lead to inconsistent reads and lost data. ");
-            log.error("QA-86204 " + ex.getMessage() + tokenRangesToHost, ex);
+            log.error("QA-86204 {}", tokenRangesToHost, ex);
```

Comment:
We should probably add the exception message here. (I think tokenRangesToHost can be large sometimes)